utils/validator.go

```diff
@@ -44,3 +46,13 @@ func extractJson(fld reflect.StructField) string {
 
 	return name
 }
+
+// registerOfficerNameValidation creates a validator rule to check names match our expected validation regex
+func registerOfficerNameValidation(v *validator.Validate) {
```

Comment:
I think this validation function would be better placed in service/insolvency_resource.go, as it is specific to the insolvency resource, rather than in a generic util function.